lib/course-db.js

```diff
@@ -272,7 +295,7 @@ module.exports.loadFullCourse = function(courseDir, logger, callback) {
                     callback(null);
                 }
                 else {
-                    loadInfoDB(course.courseInstanceDB, 'ciid', course.courseInfo.courseInstancesDir, 'infoCourseInstance.json', defaultCourseInstanceInfo, schemas.infoCourseInstance, null, course.courseInfo, logger, function(err) {
+                    loadInfoDB(course.courseInstanceDB, 'ciid', course.courseInfo.courseInstancesDir, 'infoCourseInstance.json', defaultCourseInstanceInfo, schemas.infoCourseInstance, null, course.courseInfo, logger, false, function(err) {
```

Comment:
Should we enable recursive course instances (and assessments)?

Reply:
Not sure about course instances, but definitely assessments as they can then be split by `hw`, `exam`, `quiz` on the file system without the prefix.

Reply:
Yeah, I guess I don’t see why they shouldn’t be recursive.